### PR TITLE
did homework with set and loops

### PR DESCRIPTION
**Pull request by senya2006**
check my homework and I'm waiting for some advice. Also I tried to fix my mistakes from  previous homework!

### CODE REVIEW

File: Homework_7/Task_1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+list_1 = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
+list_2 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
+
+set1 = set(list_1)
+set2 = set(list_2)
+elements = sorted(list(set1.intersection(set2)))
+print(f"Unique elements in both lists are: {elements}")
```
Comment (MarinaZelenska) [inline Homework_7/Task_1.py:7]: elements = sorted(set1.intersection(set2))

File: Homework_7/Task_3.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+numbers = [1, 2, 3, 4, 5, 6, 2, 7, 8, 9, 11]
+unique_numbers = set(numbers)
+if len(unique_numbers) == len(numbers):
+    print(f"This set: {numbers} are unique")
```
Comment (MarinaZelenska) [inline Homework_7/Task_3.py:4]: I am not sure that we need a variable  unique_numbers here
if len(numbers) == len(set(numbers)):
    print(f"This set: {numbers} are unique")

**Review by MarinaZelenska — APPROVED** [review]

**Comment by senya2006** [comment]
fixed

**Comment by senya2006** [comment]
At first I did merge and after that fixed by accident so you are not be able to see my changes but I did them.

**Pull request merged**